qiskit/algorithms/time_evolvers/trotterization/trotter_qrte.py

```diff
@@ -168,6 +168,7 @@ def evolve(self, evolution_problem: TimeEvolutionProblem) -> TimeEvolutionResult
                 self.estimator,
                 evolved_state,
                 evolution_problem.aux_operators,
+                None,
```

Comment:
Is it related to VQE?

Reply:
It's related to the `compute_eigenvalues` method that is also used in VQE. We changed the signature to take optional parameters instead of bound circuits so that the estimator cache detects the circuit (for efficiency). In trotter QRTE the circuits are not parametrized so we had to add this `None` in the parameters field.